api/working-with-extensions/bundling-extension.md

```diff
@@ -11,7 +11,37 @@ MetaDescription: Bundling Visual Studio Code extensions (plug-ins) with webpack.
 
 Visual Studio Code extensions often grow quickly in size. They are authored in multiple source files and depend on modules from [npm](https://www.npmjs.com). Decomposition and reuse are development best practices but they come at a cost when installing and running extensions. Loading 100 small files is much slower than loading one large file. That's why we recommend bundling. Bundling is the process of combining multiple small source files into a single file.
 
-For JavaScript, different bundlers are available. Popular ones are [rollup.js](https://rollupjs.org), [Parcel](https://parceljs.org), and [webpack](https://webpack.js.org/). This tutorial will focus on **webpack**, however, concepts and benefits of all bundlers are similar.
+For JavaScript, different bundlers are available. Popular ones are [rollup.js](https://rollupjs.org), [Parcel](https://parceljs.org), and [webpack](https://webpack.js.org/).
+
+## Using esbuild
+
+`esbuild` is a fast bundler that's simple to configure. To acquire esbuild, open the terminal and type:
+
+```bash
+npm i --save-dev esbuild
+```
+
+For an example of a complete extension using esbuild, check out the [test-adapter-converter](https://github.com/microsoft/vscode-test-adapter-converter).
```

Comment:
The test-adapter-converter link 404s for me.

Reply:
It doesn't exist now, but it will in about 6 hrs :)

Reply:
Woah, mad crystal ball skills

Reply:
Should I wait until vscode-test-adapter-converter is available before merging and publishing?

Reply:
Sure. I can merge it when I publish if this otherwise looks good by you.